All_home_works/hw12/migrations/script.py.mako

```diff
@@ -0,0 +1,24 @@
+"""${message}
```

Comment:
Why do we need this file?

Reply:
This file is used to generate new migration scripts.